flutter/lib/src/flutter_enricher.dart

```diff
@@ -0,0 +1,199 @@
+/// FlutterEnricher only needs to add information which aren't exposed by
+/// the Dart runtime.
+class FlutterEnricher implements Enricher {
+  FlutterEnricher(
+    this._checker,
+    this._dartEnricher,
+    this._getWidgetsBinding,
+  );
+
+  factory FlutterEnricher.simple(PlatformChecker checker) {
+    return FlutterEnricher(
+      checker,
+      Enricher(),
+      () => WidgetsFlutterBinding.ensureInitialized(),
+    );
+  }
+
+  // We can't use `WidgetsBinding` as a direct parameter
+  // because it must be called inside the `runZoneGuarded`-Integration.
+  // Thus we call it on demand after all the initialization happend.
+  final WidgetBindingGetter _getWidgetsBinding;
+  WidgetsBinding get _widgetsBinding => _getWidgetsBinding();
+  final PlatformChecker _checker;
+  final Enricher _dartEnricher;
+  SingletonFlutterWindow get _window => _widgetsBinding.window;
+  Map<String, String> _packages = {};
+
+  @override
+  FutureOr<SentryEvent> apply(
+    SentryEvent event,
+    bool hasNativeIntegration,
+    bool includePii,
+  ) async {
+    // Flutter for Web does not need a special case.
+    // It's already covered by _dartEnricher.
+
+    // First use _dartEnricher.
+    // In case we have even better information available in Flutter
+    // we override what's already given by _dartEnricher.
+    event = await _dartEnricher.apply(event, hasNativeIntegration, includePii);
+
+    // If there's a native integration available, it probably has better
+    // information available than Flutter.
+    final device =
+        hasNativeIntegration ? null : _getDevice(event.contexts.device);
+
+    final contexts = event.contexts.copyWith(
+      device: device,
+      runtimes: _getRuntimes(event.contexts.runtimes),
+    );
+
+    // Flutter has a lot of Accessibility Settings available and exposes them
+    contexts['accessibility'] = _getAccessibilityContext();
+
+    // Conflicts with Flutter runtime if it's just called `Flutter`
+    contexts['flutter_context'] = _getFlutterContext();
+
+    contexts['culture'] = _getCulture();
+
+    return event.copyWith(
+      contexts: contexts,
+      modules: await _getPackages(),
+    );
+  }
+
+  /// Packages are loaded from [LicenseRegistry].
+  /// This is currently the only way to know which packages are used.
+  /// This however has some drawbacks:
+  /// - Only packages with licenses are known
+  /// - No version information is available
+  /// - Flutter's native dependencies are also included.
+  FutureOr<Map<String, String>> _getPackages() async {
+    if (_packages.isEmpty) {
+      // This can take some time.
+      // Therefore we cache this after running
+      var packages = <String>{};
+      await LicenseRegistry.licenses.forEach(
+        (entry) => packages.addAll(
+          entry.packages.toList(),
+        ),
+      );
+
+      _packages = Map.fromEntries(
+        packages.map(
+          (e) => MapEntry(e, 'unknown'),
```

Comment:
Is `unknown` or `n/a` (for not available) better?
I kinda like `n/a` better because that's what it is. 
(It's also shorter and saves some bytes though it probably doesn't matter much)

Reply:
Then we keep `unknown`.